aim/api/types.py

```diff
@@ -41,6 +41,7 @@
                '88': 'eigrp', '89': 'ospfigp', '103': 'pim', '115': 'l2tp'}
 
 ethertype = {'0': UNSPECIFIED, '1': 'ipv4', '2': 'ipv6'}
+spmodes = {'0': 'regular', '1': 'native', '2': 'untagged'}
```

Comment:
In ACI on-line help, those 3 modes are Trunk, Access & Access Untagged. Should we just follow the naming used there instead then?
https://172.28.184.50/help/content/index.html#fv_infoAEPgStaticBindings.html

Reply:
I was looking at this:
https://172.28.184.30/doc/html/TYPE-fv-Mode.html

I wasn't sure which were the right types. The ones in the GUI are the ones you mentioned, but from an API perspective, I thought it was tricky having two that had the word "access", plus I wanted to keep the names short.

Reply:
I'm fine with it if that's what was defined in the API level.